odpf/shield/v1/shield.proto

```diff
@@ -394,18 +394,19 @@ message ListGroupsResponse {
 message Role {
   string id = 1;
   string name = 2 [(validate.rules).string.pattern = "^[A-Za-z0-9_-]+$"];
-  repeated string tags = 3;
-  repeated string actions = 4;
+  repeated string types = 3;
+  string definition_level = 4;
```

Comment:
Can you help me understand the use case of `definition_level`?

Reply:
It tells the policy namespace in which the role is applied. We can filter roles according to it.
We will also use it in dynamic schema generation.
I think "namespace" will be better key name

Reply:
yup